src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCost/ActualCost.js

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+} from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  Button,
+  Col,
+  KeyValue,
+  Modal,
+  ModalFooter,
+  Row,
+  TextArea,
+  TextField,
+} from '@folio/stripes/components';
+
+import {
+  getPatronName,
+} from '../../util';
+
+import {
+  ACTUAL_COST_TYPES,
+  ACTUAL_COST_RECORD_FIELD_NAME,
+  ACTUAL_COST_RECORD_FIELD_PATH,
+  DEFAULT_VALUE,
+  ACTUAL_COST_MODAL_DEFAULT,
+  ACTUAL_COST_DEFAULT,
+  ACTUAL_COST_PROP_TYPES,
+} from '../../../../../constants';
+
+const ActualCost = ({
```

Comment:
I would rename it to `ActualCostModal`. What do you think about it?
In my opinion, the name should say that it is a modal window.

Reply:
Fixed